internal/scaffold/cmd.go

```diff
@@ -217,8 +217,12 @@ func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
 // serveCRMetrics gets the Operator/CustomResource GVKs and generates metrics based on those types.
 // It serves those metrics on "http://metricsHost:operatorMetricsPort".
 func serveCRMetrics(cfg *rest.Config) error {
-	// Below function returns filtered operator/CustomResource specific GVKs.
-	// For more control override the below GVK list with your own custom logic.
+	// The function below returns filtered operator/CR specific GVKs. For more control, override the GVK list below
+	// with your own custom logic.
+	// Note that if you are adding third party API schemas, probably you will need to customize this implementation
+	// to not face issues such as; "Failed to list *unstructured.Unstructured:" which may occur because
+	// the operator has not permission to List the schemas added. See that when a schema is added
+	// it also may add others schemas which are required by it and then, they will not be filtered by default.
```

Comment:
Discussing errors in `cmd/manager/main.go` in depth is unnecessary. If anything, link to the relevant user guide section.

Reply:
Many users ha been facing this scenario and indeed was hard for us to figure out what happens. But I agree with you too. I will change that.